examples/with-algolia/service/serverless.yml

```diff
@@ -0,0 +1,22 @@
+service: aws-node-express-api-project
+frameworkVersion: '3'
+plugins:
+  - serverless-plugin-typescript
+  - serverless-offline
+
+provider:
+  name: aws
+  runtime: nodejs16.x
+
+useDotenv: true
+
+functions:
+  api:
+    handler: handler.handler
+    events:
+      - httpApi: '*'
+    environment:
+      ALGOLIA_INDEX: ${env:ALGOLIA_INDEX}
+      ALGOLIA_API_KEY: ${env:ALGOLIA_API_KEY}
+      STOREFRONT_API_ENDPOINT: ${env:STOREFRONT_API_ENDPOINT}
+      STOREFRONT_API_TOKEN: ${env:STOREFRONT_API_TOKEN}
```

Comment:
Given potential for confusion with the Shopify Storefront API, probably worth prefixing these env vars with `NACELLE_`.

Reply:
Updated!